plutarch-extra/Plutarch/Extra/TermCont.hs

```diff
@@ -16,3 +20,11 @@ pletC = tcont . plet
 -- | Like `pmatch` but works in a `TermCont` monad
 pmatchC :: PMatch a => Term s a -> TermCont s (a s)
 pmatchC = tcont . pmatch
+
+-- | Like `ptryFrom` but works in a `TermCont` monad
+ptryFromC :: forall b a s. PTryFrom a b => Term s a -> TermCont s (Term s b, Reduce (PTryFromExcess a b s))
+ptryFromC = tcont . ptryFrom
+
+-- | Like `ptryFom'` but works in a `TermCont` monad
+ptryFromC' :: forall a b s. PTryFrom a b => Term s a -> TermCont s (Term s b, Reduce (PTryFromExcess a b s))
```

Comment:
Why add this?

Reply:
Just so we have 1:1 correspondence between the ptryFrom and respecting TermCont methods, if you want I can scratch that.

Reply:
You never use `ptryFrom'`, thus neither `ptryFromC'`.

Reply:
got it, removed.